Buildings/Experimental/DHC/Loads/Cooling/BaseClasses/BuildingTimeSeries.mo

```diff
@@ -0,0 +1,326 @@
+    "Design cooling heat flow rate (<=0)"
+    annotation (Dialog(group="Nominal condition", enable=have_chiWat));
+  parameter Modelica.Units.SI.HeatFlowRate QHea_flow_nominal(
+    min=Modelica.Constants.eps)=Buildings.Experimental.DHC.Loads.BaseClasses.getPeakLoad(
+    string="#Peak space heating load",
+    filNam=Modelica.Utilities.Files.loadResource(filNam))
+    "Design heating heat flow rate (>=0)"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mChiWat_flow_nominal=abs(
+    QCoo_flow_nominal/cp_default/(T_aChiWat_nominal-T_bChiWat_nominal))
+    "Chilled water mass flow rate at nominal conditions (all units)"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate mHeaWat_flow_nominal=abs(
+    QHea_flow_nominal/cp_default/(T_aHeaWat_nominal-T_bHeaWat_nominal))
+    "Heating water mass flow rate at nominal conditions (all units)"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Real k(
+    min=0)=0.1
+    "Gain of controller";
+  parameter Modelica.Units.SI.Time Ti(
+    min=Modelica.Constants.small)=10
+    "Time constant of integrator block";
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.SteadyState
+    "Type of energy balance"
+    annotation (Evaluate=true,Dialog(tab="Dynamics",group="Conservation equations"));
+  parameter Boolean use_inputFilter=false
+    "= true, if pump speed is filtered with a 2nd order CriticalDamping filter"
+    annotation(Dialog(tab="Dynamics", group="Pump"));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput QReqHotWat_flow(
+    final unit="W") if have_hotWat
+    "SHW load" annotation (Placement(
+      transformation(extent={{300,-140},{340,-100}}), iconTransformation(
+      extent={{-40,-40},{40,40}},
+      rotation=-90,
+      origin={280,-340})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput QReqHea_flow(
+    final quantity="HeatFlowRate",
+    final unit="W") if have_heaLoa
+    "Heating load"
+    annotation (Placement(transformation(extent={{300,20},{340,60}}),
+      iconTransformation(extent={{-40,-40},{40,40}},rotation=-90,origin={200,-340})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput QReqCoo_flow(
+    final quantity="HeatFlowRate",
+    final unit="W") if have_cooLoa
+    "Cooling load"
+    annotation (Placement(transformation(extent={{300,-20},{340,20}}),
+      iconTransformation(extent={{-40,-40},{40,40}},rotation=-90,origin={240,-340})));
+  Modelica.Blocks.Sources.CombiTimeTable loa(
+    tableOnFile=true,
+    tableName="tab1",
+    fileName=Modelica.Utilities.Files.loadResource(
+      filNam),
+    extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic,
+    y(each unit="W"),
+    offset={0},
+    columns={2},
+    smoothness=Modelica.Blocks.Types.Smoothness.MonotoneContinuousDerivative1)
+    "Reader for thermal loads (y[1] is cooling load, y[2] is heating load)"
```

Comment:
Why do we need two columns, one being for heating, if this model is only for cooling?